tools/cppfind

```diff
@@ -2,6 +2,6 @@
 
 FUSE_CFLAGS="$(pkg-config --cflags fuse) -DFUSE_USE_VERSION=29"
 
-echo "#include <fuse.h>" | cpp ${FUSE_CFLAGS} | grep -q "${1}"
```

Comment:
`-q` means grep quits as soon as it finds a match, hence Clang complaining about being unable to write to the pipe.